sites/public/src/lib/runtime-config.ts

```diff
@@ -35,6 +35,25 @@ export const runtimeConfig = {
   */
 
   getGoogleMapsApiKey() {
+    //console.log(this.env)
+    //console.log(`GOOGLE_MAPS_API_KEY = ${this.env.GOOGLE_MAPS_API_KEY}`)
     return this.env.GOOGLE_MAPS_API_KEY
   },
+
+  getBackendApiBase() {
```

Comment:
I know this has been covered before, but the `runtimeConfig` object/package is necessary for keeping the build pipeline functional while still being able to run different environments with different values.  The backend API is probably the most important thing to have represented here since it's guaranteed to change between environments